README.md

```diff
@@ -753,6 +753,23 @@ Use the system's local time to determine the log file name as well as when the l
 
 Specifies what W3C fields are logged in the log file. This is only valid when `logformat` is set to 'W3C'.
 
+##### `authenticationinfo`
+
+Enable and disable authentication schemas. The available schemas are: anonymous, basic, clientCertificateMapping, digest, iisClientCertificateMapping, windows.
```

Comment:
Are the authentication schema names case sensitive?  The camel-casing suggests it may be,.

Reply:
Type validation prevents improper casing or spelling.